themes/theme-b2c-tailwind/src/molecules/accordion.css

```diff
@@ -1,19 +1,38 @@
-[data-store-accordion-button] {
-  @apply bg-transparent text-gray-800;
-  @apply hover:text-pink-600;
+[data-accordion-button] {
+  color: #171a1c;
```

Comment:
I do update this stylesheet. But this was not this task's purpose. So I decided to do it separately later.